src/index.js

```diff
@@ -68,11 +48,7 @@ function bindEvents (options) {
     documentCapture: () => options.onDocumentCapture(getCaptures().documentCapture),
     documentBackCapture: () => options.onDocumentCapture(getCaptures().documentBackCapture),
     faceCapture: () => options.onFaceCapture(getCaptures().faceCapture),
-    complete: () => options.onComplete(getCaptures()),
-    onError: () => {
```

Comment:
Before this change the `onError` callback was not documented and I don't think it ever got fired. I think it is important to let the developer cope with errors - if the server is down or we hit an exception.

Given this change does not remove function I think we need more thought and should deal with it in a separate issue. For example I think the iOS SDK exposes errors in an error object passed to `onComplete`.